.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Create also an .npmignore file ad list node_modules and examples and src folders in there